src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -258,15 +259,15 @@ void OrbitMainWindow::SetupMainWindow() {
   app_->SetStatusListener(status_listener_.get());
 
   app_->SetCaptureStartedCallback([this](const std::optional<std::filesystem::path>& file_path) {
-    UpdateCaptureStateDependentWidgets();
-    ClearCaptureFilters();
-
     // Only set it if this is not empty, we do not want to reset the label when loading from legacy
     // file format.
     if (file_path.has_value()) {
       target_label_->SetFile(file_path.value());
       setWindowTitle(QString::fromStdString(file_path.value().string()));
     }
+
+    UpdateCaptureStateDependentWidgets();
+    ClearCaptureFilters();
```

Comment:
Whats the reason to move this? If there is one, maybe a comment makes sense? If there none, you can ignore this comment

Reply:
enabling the menu option depends on the value of target_label_, moved UpdateCaptureState... so that it reads the correct state of the target_label_

Reply:
added a comment.

Reply:
gotcha